conf/flight_plans/rover_optitrack_PIR.xml

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE flight_plan SYSTEM "flight_plan.dtd">
```

Comment:
What does it mean "PIR" ?

I suppose that in the near future we should have ./plans/flightplans ./plans/"groundplans"/ etc?

Reply:
who knows...
PIR stands for Projet Initiation Recherche. It is not really necessary to keep this one, but it has some nice moves for holonomic rover.